app/models/document-version.js

```diff
@@ -57,16 +58,27 @@ export default Model.extend({
     });
   }),
 
-  storeAccessLevel(accessLevel) {
-    this.set('modified', moment().toDate());
-    this.set('accessLevel', accessLevel);
-    return this.save();
+  changePublicSince() {
+    if (this.get('accessLevel').get('id') === '6ca49d86-d40f-46c9-bde3-a322aa7e5c8e' && !this.get('confidential') && !this.get('publicSince')) {
```

Comment:
De id's [hier](https://github.com/kanselarij-vlaanderen/kaleidos-frontend/blob/development/app/utils/config.js) configureerbaar maken is een aanrader?

Reply:
+1, We dienen dit ook op andere plekken beginnen toe te passen.